xrayvision/Visibility.py

```diff
@@ -27,9 +33,11 @@ class Visibility(object):
 
     """
 
-    def __init__(self, uv, vis):
+    def __init__(self, uv, vis, xyoffset=(0., 0.), pixel_size=(1., 1.)):
```

Comment:
Maybe set the xyoffset and pixel size to `None` at this point as depending on where the data will from from a map or a fits file it will get set then.